romancal/pipeline/exposure_pipeline.py

```diff
@@ -78,29 +76,29 @@ def process(self, input):
 
         # open the input file
         file_type = filetype.check(input)
-        asn = None
+        if file_type == "asn":
+            asn = ModelContainer.read_asn(input)
+
         if file_type == "asdf":
             try:
-                input = rdm.open(input)
+                asn = ModelContainer(None)
```

Comment:
We don't need to initialize `asn` as a `ModelContainer` here since it'll be built from the provided asdf file on the next line.